studio/LonaStudio/Workspace/LayerListHeader.swift

```diff
@@ -20,6 +21,10 @@ public class LayerListHeader: NSBox {
         setUpConstraints()
 
         update()
+
+        subscriptions.append(LonaPlugins.current.register(eventType: .onChangeFileSystemComponents) {
+            self.setUpViews()
```

Comment:
Calling `setUpViews` here does work, but that method is a pattern we use in every component, and it generally shouldn't be called more than once. Let's make a new method that just updates the menu of the button, e.g. with:

```
button.menu(forSegment: 0)?.items = ComponentMenu.menuItems()
```

I think we should also add an `[unowned self]` to this block to make sure it gets deallocated, e.g. https://github.com/airbnb/Lona/blob/2aa1745835e2e572dc04c1d54a66d0133bc4b45d/studio/LonaStudio/Workspace/WorkspaceViewController.swift#L764-L766 I usually try to do this, but sometimes I forget, so that's why it's not everywhere in the codebase.